connect/runtime/src/main/java/org/apache/kafka/connect/runtime/health/ConnectClusterDetailsImpl.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.connect.runtime.health;
+
+import org.apache.kafka.connect.health.ConnectClusterDetails;
+
+public class ConnectClusterDetailsImpl implements ConnectClusterDetails {
```

Comment:
Still a javadoc would be nice, stating that this is a basic/simple implementation of `ConnectClusterDetails` meant to be used for ... etc

Reply:
There isn't a matching javadoc on the `ConnectClusterStateImpl` class and I'm not sure it'd be too useful to add one here. If we mention that it's explicitly for use in REST extensions that may become outdated if/when the class is meant to be used elsewhere. There's pretty sufficient detail on the behavior of the class provided in the javadocs for the interface it implements.

Reply:
Not a blocker for me, but I think any javadoc is better than no javadoc for a class or an interface. 
At the class level, the javadoc is not inherited by the base class (as when `@Override` is used in methods). So the presence of javadoc in the interface but not here does not save us the extra hoop. In any case, I mentioned since this is a small PR and I think it's a good practice (that at least I try to follow as much as possible). Not a blocker for this PR.